src/main/java/frc/robot/RobotContainer.java

```diff
@@ -86,26 +89,39 @@ private void configureButtonBindings() {
         sensitiveDriverControlCommand = new SensitiveDriverControl(driverStick);
 
     }
-    
+
     /**
      * Use this to pass the autonomous command to the main {@link Robot} class.
      *
      * @return the command to run in autonomous
      */
     public Command getAutonomousCommand() {
-        int selectorOne = 0, selectorTwo = 0;
-        if (analogSelectorOne != null) selectorOne = analogSelectorOne.getIndex();
-        if (analogSelectorTwo != null) selectorTwo = analogSelectorTwo.getIndex();
+        int selectorOne = 1, selectorTwo = 1;
+        if (analogSelectorOne != null)
+            selectorOne = analogSelectorOne.getIndex();
+        if (analogSelectorTwo != null)
+            selectorTwo = analogSelectorTwo.getIndex();
         logger.info("Selectors: " + selectorOne + " " + selectorTwo);
-        
+
         if (selectorOne == 0 && selectorTwo == 0) {
             // This is our 'do nothing' selector
             return null;
         }
-        
-        // If we had more auto options I'd add them here lol
-        
+
+
+        if (selectorOne == 1 || selectorTwo == 1) {
+        /*
+         * When the selector is set to one, the robot will run for x seconds at y left motor speed and z right motor speed
+         */
+
+      // for the inputed variables: DriveWithTime(seconds (double), left motors speed (double), right motors speed (double))
```

Comment:
This isn't really necessary since we can just `CTRL + CLICK` on the constructor.